psw/urts/enclave.cpp

```diff
@@ -43,7 +43,7 @@
 using namespace std;
 
 int do_ecall(const int fn, const void *ocall_table, const void *ms, CTrustThread *trust_thread);
-int do_ocall(const bridge_fn_t bridge, void *ms);
+int do_ocall(const bridge_fn_t bridge, sgx_enclave_id_t enclave_id, void *ms);
```

Comment:
It's backwards compatible in the sense that existing projects will compile fine if the files are regenerated.